paypal/express_checkout/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+
+class ExpressCheckoutTransaction(models.Model):
```

Comment:
Please add a docstring, explaining the purpose of this model (e.g. dedicated to store transactions for "express checkout sdk".

Reply:
Done.